### ISSUE TITLE
feature request: support for dict_tracebacks

### ISSUE THREAD

**Issue by g--** [multani/structlog-gcp#35]
Hi there! I appreciate this library as it's made it easier to integrate into stackdriver/gcp logging. One small feature request: support for [structlog.processors.dict_tracebacks](https://www.structlog.org/en/stable/api.html#structlog.processors.dict_tracebacks), or similar functionality included

I found when I had both `dict_tracebacks` and `structlog_gcp` enabled like this:

```
    structlog.configure(processors=[
        structlog.processors.dict_tracebacks,
        *(structlog_gcp.build_processors()),
    ])
```

stacktraces were rendered as dictionaries and then back into strings, so they'd show up as json strings both on the console and in gcp.

**Comment by multani** [comment]
Hey @g-- , you shouldn't need to use `structlog.processors.dict_tracebacks` at all: if you log an exception, it should be correctly rendered as a GCP-compatible log error.

See for instance this test: https://github.com/multani/structlog-gcp/blob/64b029252686ef35e35c072f9faf2b29b0310af3/tests/test_log.py#L28-L60

**Comment by g--** [comment]
That's true! But the `stack_trace` field is a string instead of a structured list of stack elements. And at the very least, I usually need to see the actual error message embedded in the stack trace. (Actually if you could pull the error message of the initial exception out as a top level element, that would be super handy!)

**Comment by multani** [comment]
The `stack_trace` will contains the plain text value of the exception.

If tou take this example:

```python
        try:
            1 / 0
        except ZeroDivisionError:
            logger.exception("division by zero")
```

It shows like this into Cloud Logging:
![](https://github.com/multani/structlog-gcp/raw/main/docs/errors.png)

And would also show the full error message with the stack trace correctly logged into Error Reporting:

![](https://github.com/multani/structlog-gcp/raw/main/docs/logs.png)

I can check if you don't specify a message to the logger if the exception name can be put instead, but you could also do:

```python
try:
    ...
except Exception as exc:
    logger.exception(str(exc))
```

> But the `stack_trace` field is a string instead of a structured list of stack elements

What do you mean by that exactly? Are you looking to get all the details of the exceptions into separated fields in the log record?

Do you have an example to share how it would look like?
